src/components/AllAlerts/index.js

```diff
@@ -29,7 +23,7 @@ class AllAlerts extends Component {
       3: 0,
       2: 0,
       1: 0,
-      0: 0
+      0: 0,
```

Comment:
remove comma

Reply:
it's due to my linter unfortunately lol!

Reply:
haha lol ok we can keep it then!